### PR TITLE
[nibeuplink] Remove org.apache.common

### PR DESCRIPTION
**Pull request by lsiepel**
- Remove dependency on apache.commons

Untested, minor refactoring from only from code perspective

### CODE REVIEW

File: bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/config/NibeUplinkConfiguration.java [hunk 1]
```diff
@@ -91,9 +90,9 @@ public void setHouseKeepingInterval(Integer houseKeepingInterval) {
 
     @Override
     public String toString() {
-        return new ToStringBuilder(this).append("user", getUser()).append("password", getPassword())
-                .append("nibeId", getNibeId()).append("pollingInterval", getPollingInterval())
-                .append("houseKeepingInterval", getHouseKeepingInterval()).append("asyncTimeout", getAsyncTimeout())
-                .append("syncTimeout", getSyncTimeout()).toString();
+        return getClass().getSimpleName() + "{ user=" + getUser() + ", password=" + getPassword() + ", nibeId='"
```
Comment (lolodomo) [inline bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/config/NibeUplinkConfiguration.java:93]: I see an unexpected simplequote just after `nibeId=`

File: bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/config/NibeUplinkConfiguration.java [hunk 2]
```diff
@@ -91,9 +90,9 @@ public void setHouseKeepingInterval(Integer houseKeepingInterval) {
...
+                + getNibeId() + ", pollingInterval=" + getPollingInterval() + ", houseKeepingInterval="
+                + getHouseKeepingInterval() + ", asyncTimeout=" + getAsyncTimeout() + ", syncTimeout="
+                + getSyncTimeout().toString() + "}";
```
Comment (lolodomo) [inline bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/config/NibeUplinkConfiguration.java:96]: No need of the `.toString()` after `getSyncTimeout()`

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/config/NibeUplinkConfiguration.java (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/config/NibeUplinkConfiguration.java:93]: Nice catch, thanks

File: bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/config/NibeUplinkConfiguration.java (hunk 2 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/config/NibeUplinkConfiguration.java:96]: yes

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**